tests/config/pytest_modules.yml

```
@@ -666,6 +666,20 @@ cellranger/vdj:
   - modules/nf-core/cellranger/vdj/**
   - tests/modules/nf-core/cellranger/vdj/**
 
+cellrangeratac/count:
+  - modules/nf-core/cellrangeratac/count/**
+  - tests/modules/nf-core/cellrangeratac/count/**
+  - modules/nf-core/cellrangeratac/mkref/**
+  - tests/modules/nf-core/cellrangeratac/mkref/**
+
```

```suggestion
  - modules/nf-core/unzip/**
  - tests/modules/nf-core/unzip/**
  
```
do you need to add this? normally all used modules are included, and your `cellrangeratac/count` module uses `UNZIP`

---

No you dont have to. It works without and other tools which use UNZIP aso do not include it in the pytest_modules.yml.